manifests/configmap-seed/kustomization.yml

```diff
@@ -5,13 +5,12 @@ resources:
   - mtls-role.yml
   - mtls-serviceaccount.yml
 patches:
-  - add-init-containers.patch.yml
-  - disable-seed-on-init.patch.yml
+  - add-sidecars.patch.yml
+  - remove-persistence-pvc.patch.yml
 configMapGenerator:
-  - name: mtls-seed-script
+  - name: mtls-restart-script
     files:
-      - ./files/seed.sh
+      - ./files/restart-mtls.sh
 images:
   - name: kiwigrid/k8s-sidecar
-    newName: drgrove/k8s-sidecar
-    digest: sha256:8799b42ad45245b63a85e0e2190914da8d32472f3cd98f057cc332ca32be98cc
+    digest: sha256:fcb3ab84daf0269e811cffa5334e3acb75abd215
```

Comment:
this shasum looks like it got cut off

Reply:
Hmm, I pulled that from the github UI, and you're right, it's not a sha256 hash.

I couldn't find the sha256 anywhere, so I ended up using a tool called kbld to get it (see update).  Do you know of a better way?